Cargo.toml

```diff
@@ -98,7 +98,7 @@ exclude = [
 
 [workspace.package]
 version = "1.5.0"
-rust-version = "1.67"
+rust-version = "1.70"
```

Comment:
Thought: I think our policy allows 1.72? We could update all the way.

Reply:
I didn't find anything useful in 1.71-1.73 (which our policy allows in two weeks)